apps/site/src/components/PageLanding/index.tsx

```diff
@@ -39,6 +41,19 @@ export default function PageLanding({ nfts }: Props) {
     }
   }, [setIsLandingPageLoaded]);
 
+  useEffect(() => {
+    const onScroll = () => {
+      if (window.scrollY < window.innerHeight / 2) setIsScrolled(false);
+      else setIsScrolled(true);
+    };
+
+    window.addEventListener('scroll', onScroll);
+
+    return () => {
+      window.removeEventListener('scroll', onScroll);
+    };
+  });
```

Comment:
@absalvaran add an empty dependency array so that this only runs once instead of every render.

https://stackoverflow.com/questions/57760842/why-would-we-use-useeffect-without-a-dependency-array